homeassistant/components/miele/sensor.py

```diff
@@ -417,6 +464,25 @@ class MieleSensorDefinition:
             ),
         ),
     ),
+    *(
+        MieleSensorDefinition(
+            types=(
+                MieleAppliance.HOB_HIGHLIGHT,
+                MieleAppliance.HOB_INDUCT_EXTR,
+                MieleAppliance.HOB_INDUCTION,
+            ),
+            description=MieleSensorDescription(
```

Comment:
What about adding also an attribute reporting the numeric value of the plate?
This could be useful to set up cards showing level as a slider

Reply:
@aturri The core team are usually reluctant to accept the use of attributes. I can see the point to expose the raw value here, but note that the numeric values are discontinous i.e. 0 os off, 110 is low power warming and 1 is lowest cooking power. You would have to make some sort of mapping regardless if you are using the raw value or the entity state string as key.